Safe placeholder guidance for personalisation

saimaghafoor opened the pull request:
Added safe placeholder guidance. Restructured content to accommodate new guidance.

Please could we add a preview, thanks.

BlessedDev commented:
<img width="1058" alt="safe-placeholder" src="https://github.com/user-attachments/assets/6f263b6c-edfb-4c6d-b27a-a2e33f84984d" />

BlessedDev commented:
<img width="939" alt="placeholder content" src="https://github.com/user-attachments/assets/3d5ad450-0c24-4e80-9e58-db8a0fab207c" />

saimaghafoor commented:
Yeah it'll need adding to the prototype in prep for the usability session on Thursday

rparke commented:
> Yeah it'll need adding to the prototype in prep for the usability session on Thursday

Cool, it all looks good to me, but we will need to make sure everything gets merged into the right place in the right order.

saimaghafoor commented:
@rparke I've updated the placeholder name from make_safe to secure now. And added the 'edit email template' page content too. So it's good to add to the prototype after you've reviewed it

saimaghafoor commented:
@rparke tagging you as I've added the correct placeholder syntax (Edit email template page commit), Thanks!
